volumes/miovision/README.md

```diff
@@ -165,7 +165,7 @@ The process in [**Processing Data from CSV Dumps**](#4-processing-data-from-csv-
 
 #### `volumes_15min_tmc`
 
-`volumes_15min_tmc` contains data aggregated into 15 minute bins. Interpolation process is no longer included in the new process now that we have enough data. In the new gap filling process, we try to identify hours of data that we can trust (aka not found in the `unacceptable_gaps` table) and set those 15min bin to NULL instead of 0. To be more specific, 
+`volumes_15min_tmc` contains data aggregated into 15 minute bins. Interpolation process is no longer included in the new process now that we have enough data. In the new gap filling process, we try to identify hours of data that we can trust (aka not found in the `unacceptable_gaps` table) and set those 15min bin to 0 instead of NULL. To be more specific, 
```

Comment:
This change is incorrect. Gaps should be stored as `NULL` so they don't get included in averages. This paragraph is conflating two things though:
- Gap finding: where there is no data because we suspect a data issue
- zero-filling: inserting 0s where there is no data and we think that is normal.

Reply:
I'm just confused about the statement:

> identify hours of data that we can trust (aka not found in the `unacceptable_gaps` table) 

which sounds like the documentation is referring to the data that we can trust doesn't have any detection issues, and therefore any zeros in those 15min bins should be 0, not NULL.

Reply:
ugggggggggggggh. You're right.